### PR TITLE
Added tests for getElementalValue of MooseVariableFE

### PR DESCRIPTION
**Pull request by joshuahansel**
Refs #11451

### CODE REVIEW

File: test/include/auxkernels/GetElementalValueAux.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+/**
+ * Tests the \c getElementalValue() function of \c MooseVariableFE.
+ */
+class GetElementalValueAux : public AuxKernel
+{
+public:
+  GetElementalValueAux(const InputParameters & params);
+
+  virtual ~GetElementalValueAux();
```
Comment (andrsd) [inline test/include/auxkernels/GetElementalValueAux.h]: I believe we do not need empty destructors any more.

**Review by andrsd — CHANGES_REQUESTED** [review]

File: test/include/auxkernels/GetElementalValueAux.h (hunk 1 above)
Comment (permcody) [reply test/include/auxkernels/GetElementalValueAux.h]: Correct, unless you are creating a new base class.

**Comment by joshuahansel** [comment]
The failed recover tests were because I used the `AuxKernel` on a nodal variable. I added a check in the constructor.

**Review by andrsd — APPROVED** [review]

**Comment by joshuahansel** [comment]
I can't see why the Rattlesnake failure would be related.

**Comment by andrsd** [comment]
I do not think it is related to your PR.

**Pull request merged**